troposphere/constants.py

```diff
@@ -158,7 +187,9 @@
 G2_2XLARGE = 'g2.2xlarge'
 G2_8XLARGE = 'g2.8xlarge'
 
+G3S_XLARGE = 'g3s.xlarge'
 G3_2XLARGE = 'g3.2xlarge'
```

Comment:
While you didn't add it, it looks like G3_2XLARGE isn't a valid option. Could you verify and remove as well?

Reply:
I was wondering about this since I couldn't find it anywhere. If you Google it there are some references to it, but none seem to be from AWS documentation. https://www.ec2instances.info/ also does not list it. I went through the "launch instance" workflow on us-east-1 and didn't see this as an option either.

I've pushed a commit to remove it.

Reply:
Thanks for the quick fix.